### ISSUE TITLE
[Feature Request]Brew cask support

### ISSUE THREAD

**Issue by ciiiii** [v2ex/launcher#4]
Maybe support brew install is better.

**Comment by livid** [comment]
Yes, the goal is to be able to provide the app via a simple install command like this:

```
brew install codelauncher
```

We will look into how we can get there.

**Comment by ciiiii** [comment]
> Yes, the goal is to be able to provide the app via a simple install command like this:
> 
> 
> 
> ```
> 
> brew install codelauncher
> 
> ```
> 
> 
> 
> We will look into how we can get there.

That's great.
